app/controllers/api/v1x0/portfolios_controller.rb

```diff
@@ -106,6 +107,14 @@ def portfolio_params
       def portfolio_copy_params
         params.permit(:portfolio_id, :portfolio_name)
       end
+
+      def group_id_array_check(uuids)
```

Comment:
This should be moved out of the controller and into a mixin.

Reply:
@syncrou which mixin should we use, should I create a new one called validation_mixin? This is not related to brace so I didn't want to add it to the rbac_mixin